README.md

```diff
@@ -55,22 +94,121 @@ through the Docker Compose
 [build](https://docs.docker.com/compose/reference/build/) command:
 
 ```
-# Build container images
 docker-compose build
 ```
 
-### Start
+## How to start the service
 
-Use Docker Compose to run service locally.
+The service can be ran using the convenience script:
 
-Additional Docker Compose files are provided for scenarios such as linking to other running services.
+```
+./scripts/start
+```
+
+Or use Docker Compose to run service locally.
 
+```
+docker-compose start
+```
+
+Additional Docker Compose files are provided for scenarios such as linking to other running services.
 Link to other services:
+
 ```
 docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up
 ```
 
-## Test structure
+> Note: This service depends on the 'ffc-pay-processing' service to be able to run correctly.
```

Comment:
Suggest providing a link to `ffc-pay-processing` repository so a reader knows what it is.